docs/api-and-integration/metadata/jira.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Jira Integration
+description: Configuring Octopus integration to Jira.
+---
+
+In addition to being able to [track metadata and work item](index.md) information through your CI/CD pipeline, Octopus can also integrate with Jira to provide progress information related to the work items during deployments.
+
+The integration to Jira is additional to the work-item tracking built in to Octopus. The configuration described below assumes you already have the work-item tracking enabled in your pipeline and packages.
+
+## Deployment Events
+
+To set the scene, the end goal of the integration is to provide Jira with updates on the progress of work-items through the pipeline. It will display information similar to the following when it receives these updates for the work-items:
+
+![Jira Deployments](jira-deployment.png)
+
+As a note, _Development_ here refers to an environment type, not an environment name. The environment name in Octopus in this example was actually Azure-Dev. See below for more detail on where this becomes important.
```

Comment:
```suggestion
Note that "Development" here refers to an environment _type_, not an environment name. The environment name in Octopus in this example was actually "Azure-Dev". See below for more detail on where this becomes important.
```

Reply:
I've swapped out the formatting for bolding in both cases as bold tends to stand out more than italics or speech marks.